posthog/management/commands/start_temporal_worker.py

```diff
@@ -63,6 +66,9 @@ def handle(self, *args, **options):
             options["client_key"] = "--SECRET--"
         logging.info(f"Starting Temporal Worker with options: {options}")
 
+        port = int(os.environ.get("PROMETHEUS_METRICS_EXPORT_PORT", 8001))
```

Comment:
nit: Couldn't this be a setting? Mostly for consistency with the rest of the arguments which come from settings.

Reply:
Ah, good point. I cargo culted it from [the gunicorn config](https://github.com/PostHog/posthog/blob/5513be7731414cb1a4ffa5a7789610005a8538ae/gunicorn.config.py#L39), but in retrospect it's probably doing it this way because Django stuff isn't available yet.